policyengine_us/variables/gov/ssa/ssi/eligibility/status/is_ssi_ineligible_spouse.py

```diff
@@ -11,4 +11,5 @@ class is_ssi_ineligible_spouse(Variable):
 
     def formula(person, period, parameters):
         spouse = person("is_tax_unit_spouse", period)
-        return spouse & ~person("is_ssi_eligible_spouse", period)
+        eligible = person("is_ssi_aged_blind_disabled", period)
+        return spouse & ~person("is_ssi_eligible_spouse", period) & ~eligible
```

Comment:
Just for stylistic consistency within the formula
```suggestion
        eligible_spouse = person("is_ssi_eligible_spouse", period)
        return spouse & ~eligible_spouse & ~eligible
```